### ISSUE TITLE
Mekanism Blocks getting deleted

### ISSUE THREAD

**Issue by bagaznal** [Y0ungSandwich/Mech-and-Magic#132]
Okay, so I created a new Mech & Magic 1.0.6 Server and played on it for a couple days with my friends. The first day, me and my friend realized that all Mekanism machines, when you hovered over them, they said that they were "Enrichment Chambers." A little weird, but we didn't care at the time, nothing was actually wrong with them. Today however, things started going wrong. Whenever you place any Machines that are labeled "Enrichment Chamber" in the same chunk but are not the same machine, all "Enrichment Chambers" get deleted. This occurs roughly anywhere from 5-30 minutes after you place them. Note, that its only blocks labeled as "Enrichment Chambers", which seems to be most machines from Mekanism. the only notable exception is the digital miner, which doesn't get shown as an Enrichment Chamber. What's weird is I'm not even getting any errors in my server prompt, not a single event ever shows up as having happened. After this, I went on my single player world and had the same issue, which tells me that it probably has to do with the configs or libraries. I do know that when I went to download the server for 1.0.6, it wouldn't initially launch because it was missing some class files, so I deleted the library and dragged the one from version 1.0.5hf over, which then worked. Any clue as to whats going on? I would rather not have to delete my world if I don't have to.
![2016-12-28_18 32 31](https://cloud.githubusercontent.com/assets/1657259/21533784/45d55ecc-cd2c-11e6-8c81-03072a4a431e.png)

What I've tried:
Downgrading Mekanism on Server & Client
Re-downloading Server & Client

Unfortunately, I don't have the logs to send you since I re-downloaded everything. I'll try and reproduce the result and send those in a bit.

**Comment by Cuppycake** [comment]
It's nice to have another server owner in the works.  Mekanism is a bit buggy.  It's a slow updating mod.  We're experiencing some of this on the official server as well.  Are you running straight Forge on the server or Forge and sponge?

**Comment by bagaznal** [comment]
Straight Forge at the moment.

**Comment by bagaznal** [comment]
I'm trying to figure out whats happening though; Cause I realized after the post that our Basic Smelting Factory also disappeared with the "Enrichment Chambers", or one Osmium Compressor, one Crusher, and two logistical sorters. but this doesn't follow what I said above, so I have no clue. Also, it seems like whatever it is, it's doing it based on chunks, cause if one of those items gets deleted in that chunk, the rest do as well. The items outside that chunk seem unaffected however.

**Comment by Cuppycake** [comment]
Everyone on my sever using Mekanism is saying waila used to show Mekanism as one item but now on 1.0.6 it's back to normal.  

Were running sponge so glad to see its not a sponge issue.  I believe this is solely on Mekanism being garbage in 1.10.2.

**Comment by bagaznal** [comment]
Okay, then maybe I don't have to worry about that part. What I'm trying to figure out is the mysterious disappearance of my Mekanism machines in certain chunks. It happened 3 times that I can tell, twice with me alone on the server and once with my friend alone on the server. The first time, it got rid of our Osmium Compressor, Crusher, Enrichment Chamber, Metallurgic Infuser and a couple Logistic Sorters. That was when my friend was on alone. Next, I got on and he got off (he respawned  and placed the items, I told him to since he didn't actually get rid of it). While I was breeding my chickens, it deleted all of them again, and I replaced them again. The third time it effected another chunk, and it deleted a few Logistical Sorters.

What's odd, is generally when something like that happens, you get something within the server prompt saying something screwed up, but there was absolutely _nothing_ where it should have been. I just want to make sure my Mekanism stuff is safe before I make any design centered around it =P
